src/commands/releases.rs

```diff
@@ -67,13 +65,7 @@ impl<'a> ReleaseContext<'a> {
 pub fn make_app<'a, 'b: 'a>(app: App<'a, 'b>) -> App<'a, 'b> {
     app.about("Manage releases on Sentry.")
         .setting(AppSettings::SubcommandRequiredElseHelp)
-        .org_arg()
-        .arg(Arg::with_name("project")
-            .hidden(true)
-            .value_name("PROJECT")
-            .long("project")
-            .short("p")
-            .validator(validate_project))
+        .org_project_args()
```

Comment:
`org_project_args()` is defined as `self.org_arg().project_arg()` where `project_arg()` is

```
clap::Arg::with_name("project")
                .value_name("PROJECT")
                .long("project")
                .short("p")
                .validator(validate_project)
                .help("The project slug"),
```

So it's 1:1 change.